mobile/lib/pages/about_us_page.dart

```diff
@@ -0,0 +1,108 @@
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+import 'package:radioaktywne/components/utility/custom_padding_html_widget.dart';
+import 'package:radioaktywne/components/utility/ra_progress_indicator.dart';
+import 'package:radioaktywne/components/utility/refreshable_fetch_widget.dart';
+import 'package:radioaktywne/extensions/extensions.dart';
+import 'package:radioaktywne/pages/ra_error_page.dart';
+import 'package:radioaktywne/resources/fetch_data.dart';
+import 'package:radioaktywne/resources/ra_links.dart';
+import 'package:radioaktywne/resources/ra_page_constraints.dart';
+import 'package:radioaktywne/resources/resources.dart';
+
+import '../models/about_us_info.dart';
```

Comment:
I'd suggest using package imports, like in the rest of the project:
```suggestion
import 'package:radioaktywne/models/about_us_info.dart';
```